src/ecrc-frontend/src/components/base/declaration/Declaration.css

```diff
@@ -0,0 +1,29 @@
+.declareTitle {
+  width: 100%;
+  height: 36px;
+  background: inherit;
+  background-color: rgba(56, 89, 138, 1);
+  box-sizing: border-box;
+  border-width: 3px;
+  border-style: solid;
+  font-family: "myriad-pro", "sans serif";
```

Comment:
no custom fonts, we are using bcsans throughout the application

Reply:
ok.